notebooks/vertex_genai/solutions/gen_ai_evaluation_service.ipynb

```diff
@@ -0,0 +1,812 @@
+{
```

Comment:
<p>Which supported metrics falls into the Embedding-based metrics?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/552/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Added them</p>